src/com/oltpbenchmark/Results.java

```diff
@@ -171,14 +162,8 @@ public void writeAllCSVAbsoluteTiming(List<TransactionType> activeTXTypes, Print
         for (Sample s : latencySamples) {
             double startUs = ((double) s.startNs / (double) 1000000000);
```

Comment:
I think having start time might be good too. We could leave it there for now IMO.

Reply:
Done.